### PR TITLE
Added more details to introduction of component

### PR DESCRIPTION
**Pull request by fortepc**
Added more details specifically for the context of a user who may have never heard of Zillow or Zestimates to better understand what this sensor does.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by fortepc — COMMENTED** [review]
Updated something I forgot to finishing updating.

**Comment by fortepc** [comment]
Also, the image embedded is linked to an imgur link, not sure how to get that image hosted inside of github for proper embedding, or if I even have the necessary permissions to do so.

**Comment by fabaff** [comment]
Images can't be uploaded with the GitHub editor.

**Comment by fortepc** [comment]
So is having them posted on imgur temporarily the right thing to do then?

**Comment by frenck** [comment]
@fortepc No, you should add them to this PR.

**Comment by fortepc** [comment]
I don't know how to do that...

**Comment by frenck** [comment]
@fortepc Browse to:

https://github.com/fortepc/home-assistant.github.io/tree/patch-5

Browse to the location where you want to upload a file and hit the "Upload files" button.
image

**Comment by fortepc** [comment]
Okay... I think I got it @frenck

File: source/_components/sensor.zestimate.markdown [hunk 1]
```diff
@@ -56,3 +58,7 @@ These attributes are available:
 - Address
 - Currency
 - Amount
+
+Example screenshot 
+
+![image](home-assistant.github.io/source/images/components/zestimate/zestimateexample.png)
```
Comment (frenck) [inline source/_components/sensor.zestimate.markdown]: The image inclusion is still invalid. See:

<https://deploy-preview-6678--home-assistant-docs.netlify.com/components/sensor.zestimate/>

An example from the docs on how:
<https://raw.githubusercontent.com/home-assistant/home-assistant.io/next/source/_docs/installation.markdown>

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by fortepc — COMMENTED** [review]
Okay.. so used the html instead of markdown img embed, and also switched to relative paths instead of direct link to match the example provided above.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for all the effort @fortepc!

✅ Approved 🎉 

Happy Hacktoberfest 🎉

**Pull request merged**